src/settings/watchers/FontWatcher.ts

```diff
@@ -64,6 +64,17 @@ export class FontWatcher implements IWatcher {
     };
 
     private setSystemFont = ({ useSystemFont, font }) => {
-        document.body.style.fontFamily = useSystemFont ? font : "";
+        if (useSystemFont) {
+            // Make sure that fonts with spaces in their names get interpreted properly
+            document.body.style.fontFamily = font
+                .split(',')
+                .map(font => {
+                    font = font.trim();
+                    if (!font.startsWith('"')) font = '"' + font;
+                    if (!font.endsWith('"')) font = font + '"';
+                    return font;
+                })
+                .join(',');
+        }
```

Comment:
Do we not need to clear it in the `else` case, the original code would write `""` in that case